test/dapp/public/index.html

```diff
@@ -6,13 +6,20 @@
     <title>Local wallet test</title>
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <script src="ethers-5.7.2.umd.min.js"></script>
+    <style>
```

Comment:
Not a big deal, just wondering, why?

Reply:
the default font was bugging me haha. Otherwise, no real reason.